azure/storage/blob/appendblobservice.py

```diff
@@ -110,11 +110,13 @@ def __init__(self, account_name=None, account_key=None, sas_token=None,
             request session. See
             http://azure.microsoft.com/en-us/documentation/articles/storage-configure-connection-string/
             for the connection string format.
+        :param int socket_timeout:
+            If specified, this will override the default socket timeout.
```

Comment:
> If specified, this will override the default socket timeout. [](start = 11, length = 61)

Can you just add for all of these comments:
The timeout specified is in seconds.
See DEFAULT_SOCKET_TIMEOUT in _constants.py for the default value